calico/getting-started/kubernetes/managed-public-cloud/eks.mdx

```diff
@@ -189,7 +189,7 @@ Before you get started, make sure you have downloaded and configured the [necess
 1. Install version {{releaseTitle}} of the {{prodname}} operator and custom resource definitions.
 
     ```batch
-    helm install calico projectcalico/tigera-operator --version {{releaseTitle}}
+    helm install calico projectcalico/tigera-operator --version {{releaseTitle}} --namespace tigera-operator
```

Comment:
I think we need `--create-namespace` as well here

Or an additional step to create the namespace: https://docs.tigera.io/calico/latest/getting-started/kubernetes/helm#install-calico